src/smitaVetal/Assignment25.java

```diff
@@ -0,0 +1,56 @@
+/*Assignment - 25 : 20th Sep'2022
+
+Program 1: Print all the repeated word in given string.
+String str = "Maulik Puja Hindavi Shubham Hindavi Shubham Titiksha Swati";
+output : Hindavi 
+         Shubham 
+
+Program 2: Print all the non repeated word in given string.
+String str = "Maulik Puja Hindavi Shubham hindavi Shubham Titiksha Swati";
+output : Maulik 
+         Puja
+         Titiksha
+         Swati */
+
+package smitaVetal;
+
+public class Assignment25 {
+
+	void getRepeatedWords(String name) {
+		System.out.println("Input string is " + name + "\n\nRepeated words from given string are ");
+		String word[] = name.split(" ");
+		for (int i = 0; i < word.length; i++) {
+			for (int j = i + 1; j < word.length; j++) {
+				if ((word[i]).equals(word[j]))
+					System.out.println(word[j]);
+			}
+		}
+	}
+
+	void getNonRepeatedWords(String name) {
```

Comment:
for this you can use, indexOf() & lastIndexOf() method, if both method return same values which mean word is unique.